### DISCUSSION TITLE
Astroid v2.6.0 needs testing

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#193] [General]
Hi @Chacapamac, @Joomlaplates, @rinka88, @imarklee, @GHSVS-de

I am working for Astroid v2.6.0. This version will support child-template. I need your help to test it before release. Please help me follow this document to test child-template on your website https://ltheme.com/joomla-4-child-templates/

How to test v2.6.0:

- Download/Install developer branch https://github.com/templaza/astroid-framework/archive/refs/heads/developer.zip
- Download/Install One-template https://github.com/templaza/astroid-framework/archive/refs/heads/one-template.zip or Zero-template https://github.com/templaza/astroid-framework/archive/refs/heads/zero-template.zip
- Follow this document if you don't know how to create child-template for test child-template on your website https://ltheme.com/joomla-4-child-templates/

Thank you for your co-operation. I'm looking forward to hearing from you.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
@sonvnn  — I Sonny, hope everything is good!

**I Just follow your Steps** — This is what I get so far _( Newbie at Joomla 4 Child Template )_

—————————————————————————————————————
_**• Note:** Here I add a another Template Style (I’m not sure if really needed, but just try to see what Happen_

![1-Naming](https://user-images.githubusercontent.com/1907276/191783858-708b1cb5-b1d5-470c-abf6-9ab933934418.jpg)
—————————————————————————————————————

_**• Note:** This is the child Template follow by the included files_

![2-child-created](https://user-images.githubusercontent.com/1907276/191783914-b05c72e9-8c77-4035-b996-59d03d2b458a.jpg)

![Screen Shot 2022-09-22 at 9 49 11 AM](https://user-images.githubusercontent.com/1907276/191784040-1add0aa6-da1c-41c0-9ada-fd1c53ca6a2c.jpg)

—————————————————————————————————————
_**• Note:** This is the 2 Layouts created_

![3—2-template-layout-created](https://user-images.githubusercontent.com/1907276/191783950-e7f93698-bc62-4a8d-9c76-339dcede685f.jpg)

**Reply by Chacapamac** [reply]
**Now I try to use**
![try-to-open](https://user-images.githubusercontent.com/1907276/191787407-0817c201-ec11-4d6f-ba14-ea4405bc2a25.jpg)
**as my Default Layout**

**I get this????**
@sonvnn  — _I hide my server structure_
![bug](https://user-images.githubusercontent.com/1907276/191789067-6a7a95d0-abc5-4b54-916d-5b6a9f991494.jpg)

**Reply by Chacapamac** [reply]
I have the same warnings on both Child Template Layout.
**No errors/warnings when Astroid Debug Off**

**Comment by sonvnn** [comment]
@Chacapamac 
I fixed issue on https://github.com/templaza/astroid-framework/commit/53d2ddf6b41b9842c7eb8df9672a1629d059a87a
Please help me re-download developer branch and re-test again.

Thank you very much.

Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Just Install, working perfectly!

**No more Warnings** @sonvnn 

Good day!

**Reply by sonvnn** [reply]
Thank you very much my friend. Please help me test with Joomla 3, too.

Best Regards,
Sonny

**Comment by joomlaplates-de** [comment]
I´ve tested everything too and its working.
But we need more testing, more time. I dont wanne get in trouble because there are many options to test.

**Reply by sonvnn** [reply]
Sure! Please help me test with your old templates which is not support child-template. And Joomla 3, too

**Reply by joomlaplates-de** [reply]
Hi Sonny, I am testing it on Joomla 3 with old themes and until now everything seems to working like it should.
The new Astroid moves the folder //astroid//  to Joomla media folder and my own //scss// and //css// are still working.
Presets are working well too under PHP 8.1.
Only some //Deprecated// messages but its about Joomla 3 with PHP 8.1 and has nothing todo with Astroid.
Have a nice weekend

**Comment by Chacapamac** [comment]
After Installing the Dev branch and templates on Joomla 3 

— I get a white page on frontend with Error — 

`Error: 0 custom/variable_overrides file not found for @import: line: 1, column: 69`

Got to go soon, I will reinstall and check, later.

**Reply by Chacapamac** [reply]
**Just reinstall** 
Joomla 3.10.11
PHP: 8.1.10
Template zero
Astroid Package 2.5.16
Astroid Plugin 2.6.0

**UPDATE FRAMEWORK:**
**Framework 2.5.20** = No errors, no problems

**UPDATE TEMPLATE ZERO:**
**Template Zero 2.5.19 ** = Same errors, same problems — Admin work but Frontend white page with error
`Error: 0 custom/variable_overrides file not found for @import: line: 1, column: 69`

http://efoilsurf.graf-test.ca/

**Reply by sonvnn** [reply]
Please help me clear Astroid cache and try it again.

**Reply by Chacapamac** [reply]
@sonvnn — I clear all caches, browser, Joomla & Astroid and verify by ftp. In fact no cache files are generated anywhere when visiting the frontend. **Same error**.
**Happen when I update to the Template zero in your above link** —> Zero-template https://github.com/templaza/astroid-framework/archive/refs/heads/zero-template.zip

**I have this in the error browser console** —>  _500 (Internal Server Error)_
**I have this in my server error log** —> _[MY_IP:52329-3#APVH_www.efoilsurf.graf-test.ca] Abort request processing by PID:7445, kill: 1, begin time: 11, sent time: 11, req processed: 0_

Don’t know if related ???? —> I do have a Template Zero > scss > custom > **variable_overrides.scss** file _(That I add myself and working perfectly with older version of Template Zero)_
**Error stay even if I trash it.....**
